plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -415,6 +422,210 @@ createWidget("discourse-poll-info", {
   }
 });
 
+function fetchGroupedVoters(data) {
+  return ajax("/polls/grouped_poll_results.json", { data }).catch(error => {
+    if (error) {
+      popupAjaxError(error);
+    } else {
+      bootbox.alert(I18n.t("poll.error_while_fetching_voters"));
+    }
+  });
+}
+
+function transformUserFieldToLabel(fieldName) {
+  let newName = fieldName;
```

Comment:
guess you could replace directly on `fieldName` here, that intermediate assignment doesn't seem necessary